src/lowering/desugar.ml

```diff
@@ -92,8 +92,12 @@ and exp' at note = function
       (varP v) (varE v) ty).it
   | S.ObjBlockE (s, dfs) ->
     obj_block at s None dfs note.Note.typ
-  | S.ObjE efs ->
+  | S.ObjE (efs, []) ->
     obj note.Note.typ efs
```

Comment:
Later this can become `obj_extend note.Note.typ efs []` and `obj` then becomes redundant.